pkg/ctl/install/cmd.go

```diff
@@ -0,0 +1,100 @@
+package install
+
+import (
+	"context"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+)
+
+// Command will create the `install` commands
+func Command(flagGrouping *cmdutils.FlagGrouping) *cobra.Command {
```

Comment:
Yes.

Reply:
@errordeveloper why? also, that train passed in #1064

Reply:
> Yes, and it must have just one function.
Why?

Reply:
That is the current convention, do you disagree?

Reply:
More specifically, each handler lives in separate file. All business logic is separated, top-level files only handles registration.

Reply:
I don't see a reason to do this in this case, since there is only a subcommand.

Reply:
I think it will help new joiners and external contributors that we follow the same convention in this case. What do you think?

Regarding the business logic, moving it to it's own package makes sense from a design and testability perspective. If we do that in another PR that is fine with me.

Reply:
Alright, I will move it to `pkg/addons/flux`as per #1076